purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -430,15 +430,15 @@ class Purchases internal constructor(
      * Purchase a [StoreProduct]'s [PurchaseOption] upgrading from a previous product.
      * @param [activity] Current activity
      * @param [storeProduct] The StoreProduct of the product you wish to purchase
-     * @param [purchaseOption] Your choice of purchase options available for the StoreProduct
+     * @param [purchaseOption] For subscriptions only. Your choice of purchase options available for the StoreProduct
```

Comment:
We also need to decide how this will look for Amazon... Whether they will have any purchaseOption or not...